src/Helpers/APIHelper.js

```diff
@@ -118,7 +118,21 @@ export function getCveIdsList(apiProps) {
         'advisory_available'
     ];
 
-    let parameterArray = constructParameters(apiProps, parameterNames);
+    /* If a user has Edge systems in their account, we need to convert the affecting param to be in affectingHostType
+    * So, we use hybrid system filter to determine this.
+    * - If the user have both Conventional & Edge systems, we convert the affecting params to be in affectingHostType params
+    * because affecting is no longer being used in the backend in this case.
+    * - If the user has only Conventional systems, the backend still uses affecting param, so there is no need for the conversion.
+    *
+    * Because affecting param is no longer being used in the backend in the first case,
+    * sending it in the request would make the request fail, or in case the request does not fail, it would return bad results
+    */
```

Comment:
This is more something that could be put into the commit message or the PR.

The code itself should be expressive enough to not have to explain it.

Reply:
I will add it to the commit message 👍🏼